core-services/release-controller/_repos/ocp-4.6-rhel8.repo

```diff
@@ -69,7 +69,7 @@ failovermethod = priority
 
 [rhel-8-codeready-builder-rpms]
 name = rhel-8-codeready-builder-rpms
-baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/codeready-builder/os/
+baseurl = https://cdn.redhat.com/content/eus/rhel8/8.2/x86_64//codeready-builder/os/
```

Comment:
There's an extra slash there after the arch.

Reply:
Thanks for catching that!